### ISSUE TITLE
[QUESTION] GEOS-Chem CO2 Simulation

### ISSUE THREAD

**Issue by idfarhan** [geoschem/geos-chem#570]
[geos_chem.zip](https://github.com/geoschem/geos-chem/files/5684634/geos_chem.zip)

Hi everyone,

I am trying to simulate the CO2 from GEOS Chem from AWS. I have successfully simulated it but in the output, more than 95% of the CO2 values are between 370 to 380 ppm. I am afraid if I am making some mistakes because these values are much lower than expected. The detail about the simulation is given by:

1. I launched AWS with a GEOS-Chem image following the instructions given in the youtube video by official GEOS-Chem channel.

2. I selected the time period of simulation from 01 March to 30 July 2019 (3 hours interval, instantaneous, and with a frequency of 1 day).

3. I compiled the model and dryran the simulation and save the dryrun.log file.

4. Then I downloaded the required data from aws using log.dryrun file. All the data downloaded without any error. 

5. Finally, I ran the model and it ran flawlessly and generated output.

Is there something I am missing? For more details, I am attaching some files here. 

Please note that I converted the CO2 output from mol/mol to ppm by multiplying the values with 1e+06.

Kind regards

**Comment by yantosca** [comment]
Thanks for writing @idfarhan.  I think this might be related to the spin-up.  The sample restart files that ship with GEOS-Chem should not be used to start a production simulation... instead you should spin up your simulation for a long-enough time period so that the signature of the initial conditions from the restart file are no longer present in the species concentration.  Also, the sample restart file was probably created with emissions data that do not correspond to the latest-available inventory.

Because CO2 is a long-lived species, you might need to spin up for about 5-10 years.  You can use the same year of met-data over and over again for the spin-up.  Then take the restart file at the end of the spin-up period and use that to start your production run.   You can also check the restart files at the end of each spin-up year to see if the CO2 concentrations are more in line with the present values.

**Comment by cdholmes** [comment]
My group uses this procedure to initialize CO2 simulations, which expands on what Bob said.
1. Look up the global marine surface CO2 concentration from NOAA for a date when you will start your simulation (a). This date should be at least 2-5 years before any date that you want to analyze.
2. Calculate the global marine surface CO2 concentration in your restart file (b).
3. Multiply your CO2 restart file by a constant factor of a/b so that your restart file now has the correct surface CO2 concentration.
4. Spin up your simulation for 2-5 years to before starting analysis. 

You can also send an email request to the carbon working group to inquire if someone already has a good CO2 restart file for your date of interest.

**Comment by idfarhan** [comment]
Thank you for the explanation. I am working on it. Hope it will work.

**Comment by idfarhan** [comment]
@cdholmes Sir, we arranged flight campaigns to collect CO2 in situ data from March 10 to March 20, 2019 and we intend to compare the vertical profiles. Do you have any CO2 restart file for this duration or 2 years before these dates?



Regards

**Comment by idfarhan** [comment]
Thank you all. A member from CO2 working group shared the restart file. Issue seems resolved.

Regards

**Comment by yantosca** [comment]
Great.  I'll close out this issue.  Feel free to open a new issue if you are still having problems w/ the CO2 simulation.

**Comment by AuroraHu111** [comment]
> My group uses this procedure to initialize CO2 simulations, which expands on what Bob said.
> 
> 1. Look up the global marine surface CO2 concentration from NOAA for a date when you will start your simulation (a). This date should be at least 2-5 years before any date that you want to analyze.
> 2. Calculate the global marine surface CO2 concentration in your restart file (b).
> 3. Multiply your CO2 restart file by a constant factor of a/b so that your restart file now has the correct surface CO2 concentration.
> 4. Spin up your simulation for 2-5 years to before starting analysis.
> 
> You can also send an email request to the carbon working group to inquire if someone already has a good CO2 restart file for your date of interest.

Hi Christopher,

What is the email address of the Carbon Working Group? Thanks!

Best regards,
Liting

**Comment by cdholmes** [comment]
@AuroraHu111, The working group contacts are here: https://geoschem.github.io/working-groups.html
